lib/canadianVehicle.ts

```diff
@@ -60,21 +60,44 @@ export default class CanadianVehicle extends BaseVehicle {
   }
 
   async unlock(): Promise<any> {
-    logger.info('Begin lock request');
+    logger.info('Begin unlock request');
     const preAuth = await this.getPreAuth();
     const response = await this._request(this.endpoints.unlock, {
       pAuth: preAuth
     });
     return response.body;
   }
 
-  async start(config: StartConfig): Promise<any> {
-    logger.info('Begin start request');
+  async start(
+    airCtrl: Boolean,  // climatisation
+    heating: Boolean,  // front defrost, airCtrl will be on
+    defrost: Boolean,  // side mirrors & rear defrost
+    airTemp: number | null  // temp for clim and heating
+    ): Promise<any> {
+
+    var body =  
```

Comment:
`const`, var is from the stone age of JS.

Reply:
uhm, funny... I put it _var_ not _const_, because I am inserting data later : 
`      body.hvacInfo[airTemp] = {value: airTempValue,unit:0,hvacTempType:1}
`

but apparently you can modify a const object in typescript :/